containers/hugging-face-inference/terraform/hf-models.json

```diff
@@ -0,0 +1,46 @@
+{
+    "llama" : [
+        {
+            "file": "llama-2-7b.Q2_K.gguf",
+            "source" : "https://huggingface.co/TheBloke/Llama-2-7B-GGUF/resolve/main/llama-2-7b.Q2_K.gguf",
+            "size_gb": "2.83",
+            "ctn_endpoint": "paste container endpoint here"
```

Comment:
**issue(improvement):** we can template this file with Terraform as part of the deployment. You can see an example of templating container URLs in [this example](https://github.com/scaleway/serverless-examples/tree/main/containers/nginx-cors-private-python) where we template a shell script.